### ISSUE TITLE
No audio on iOS 10 (and most likely all other versions).

### ISSUE THREAD

**Issue by JoshuaSand0val** [endrift/gbajs#50]
It's as simple as it sounds. No sound on iOS 10 (and most likely below). The game being run was "Mario and Luigi: Superstar Saga".

**Comment by JoshuaSand0val** [comment]
Okay, however, it wasn't listed. I'm going to do some research into the bug/issue and I'll notify you if I find a potential fix.

**Comment by JoshuaSand0val** [comment]
I know this is relatively quick after my other reply, but I've attached a link to the post. It seems iOS by default mutes the Audio API until some user input. “Safari on iOS 6 effectively starts with the Web Audio API muted. It will not unmute until you attempt to play a sound in a user input event.”

http://stackoverflow.com/questions/12517000/no-sound-on-ios-6-web-audio-api/12569290

**Comment by endrift** [comment]
Yup, I know about that, but it seems iOS 10 broke that. Stuff I know used to work on iOS no longer does.
